classes/extSuperDirtStartup.sc

```diff
@@ -14,7 +14,10 @@
 		// boot the server and start SuperDirt
 		server.waitForBoot {
 			~dirt = SuperDirt(numChannels, server);
-			~dirt.loadSoundFiles(path);   // load samples (path can be passed in)
```

Comment:
thank you! I did this this morning, it should already be in master here: https://github.com/musikinformatik/SuperDirt/commit/f96f5fcfe69716c3595726e72d495f8c0fff2580